scripts/js/lib/api/processHtml.ts

```diff
@@ -439,9 +439,20 @@ export function maybeSetModuleMetadata(
   $: CheerioAPI,
   $main: Cheerio<any>,
   meta: Metadata,
+  isReleaseNotes: boolean,
   options: { isCApi: boolean },
 ): void {
-  const modulePrefix = options.isCApi ? "group__" : "module-";
+  if (options.isCApi) {
+    // Every page in the C API should be displayed as a module except the index
+    // and the release notes.
+    const topHeadingText = $main.find("h1").first().text();
+    const isIndex = topHeadingText === "Qiskit C API (qiskit.h)";
```

Comment:
Rather than hardcoding this, is it possible to look at the page name? It's brittle to have a magic value like this which the Qiskit SDK team could change without us realizing.

Generally, consider refactoring this to separate out C from Python, given how different they are. Like rename it to be `maybeSetPythonModuleMetadata` and `setCMetadata` etc. The two implementations are now quite different.

Reply:
Filename is a good call, I didn't forgot that's what we did for release notes. Done both, thanks